managers/opportunity/index.js

```diff
@@ -39,20 +39,134 @@ class opportunityManager {
     }
   }
 
+  // http://localhost:3030/opportunity?limit=4&page=2, Example of how to use the Get Method
+
+  //  page => Means the Current Page that the User Wants
+  //  limit => The count of pages that should be rendered starting from the given page
+
+  //  For the above specified example , we Get Pages 2,3,4,5 (having Opprtunity_id as 1,2,3,4 and respectively) .
```

Comment:
If page = 2 and limit is 4, 
then we should get opportunities with id `4,5,6,7` (assuming ids are 0,1,2,3,4,5,6,7, etc) , Isn't it? 
@Kashish0423

Reply:
Hello @Manvityagi  mam sorry for the late response , now  I get your point about the page and the limit ,  with the example you explained above , I will try to get it working as soon as possible  .

Reply:
@Kashish0423 Yes it has been changed to level 3, Waiting for your PR.

Reply:
Yes Mam , on it and  trying to get it done asap